content/en/synthetics/multistep.md

```diff
@@ -44,11 +44,11 @@ Multistep API tests can run from Datadog [managed locations][4] and [private loc
 
 Select the **Locations** to run your Multistep API test from: Multistep API tests can run from both [managed][4] and [private locations][5] depending on whether you are willing to run the test from outside or inside your network.
 
-### Define requests
+### Define steps
 
-Hit **Create Your First Request** to start designing your test's requests.
+To create an HTPP request step, click **Create Your First Step**.
 
-{{< img src="synthetics/api_tests/create_request.png" alt="Create your Multistep API test requests"  style="width:90%;" >}}
+{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests" style="width:90%;" >}}
```

Comment:
Ok weird feedback but that image seems more blurry than the existing one, any idea why?

Reply:
This image is definitely sharper than the original, I'm bump up the image width so it appears larger!

Reply:
```suggestion
{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests" style="width:100%;" >}}
```